src/lti_toolbox/models.py

```diff
@@ -25,6 +25,13 @@ class LTIConsumer(models.Model):
         blank=False,
     )
 
+    homepage = models.URLField(
```

Comment:
why not just a "domain" field like in github.com/openfun/marsha? Is it really relevant to record a full url?
As for the name, is it really a homepage since it has a path : would "url" be a better field name?

Reply:
I think that storing only the `domain` is an unnecessary loss of information (like the scheme). If we want to obtain a link to the consumer, this requires us to generate a URL based on conventions that can be a source of error.
URL are more precise and easy to parse, so I prefer to store them.

About the name, i'll change it to `url` as suggestied. I had my use case in mind when I chose this name :)

Reply:
Good point let's go!